packages/contracts/package.json

```diff
@@ -13,7 +13,7 @@
   "types": "dist/index",
   "license": "MIT",
   "scripts": {
-    "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint",
+    "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint && yarn generate-markdown",
```

Comment:
I wasn't sure of the best place to run it but figured it should happen here

Reply:
Probably should be part of the deploy script.

Reply:
moved 👍